tooltester/src/main/java/io/dockstore/tooltester/helper/S3CacheHelper.java

```diff
@@ -9,8 +9,8 @@ public static String mapRepositoryToCache(String repository) {
         switch(repository) {
         case "github.com/briandoconnor/dockstore-workflow-md5sum":
             return "ga4gh-dream/md5sum";
-        case "github.com/dockstore/hello_world":
-            return "ga4gh-dream/hello_world";
+//        case "github.com/dockstore/hello_world":
```

Comment:
Intentional?

(the usual, code comment if it might be used some day, delete if not)